src/server/editorServices.ts

```diff
@@ -3632,6 +3640,7 @@ namespace ts.server {
             if (proj.typeAcquisition.enable === undefined) {
                 proj.typeAcquisition.enable = hasNoTypeScriptSource(proj.rootFiles.map(f => f.fileName));
             }
+            proj.typeAcquisition.inferTypings = this.hostConfiguration.typeAcquisition?.inferTypings ?? true;

```

Comment:
This seems incorrect.. host.typeAcquisition shouldnt mean for all projects.. external project can configure its on options so it shouldnt use default configured thing at all

Reply:
I did intend for it to be global. For VS this would be under tools > options. Is there a more preferred way of configuring external projects?

Reply:
Should `openExternalProjects` handle the setting `typeAcquisition` for external projects and then only `inferred` projects get the setting from the host configuration?

Reply:
IMO you would at least want to allow an external project to specify what it wants and fall back to the host config.

Reply:
The setting should be only for inferred projects rather than all.. You should look at compilerOptions for inferred project as they can also differ with projectRoot. You would want similar.